src/main/java/com/zerobase/communityproject/controller/CommentController.java

```diff
@@ -0,0 +1,36 @@
+package com.zerobase.communityproject.controller;
+
+import com.zerobase.communityproject.model.request.CommentRequest;
+import com.zerobase.communityproject.service.CommentService;
+import com.zerobase.communityproject.service.MemberService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("comment")
+@RequiredArgsConstructor
+public class CommentController {
+
+    private final MemberService memberService;
+    private final CommentService commentService;
+
+    @PostMapping
+    public ResponseEntity<?> newComment (@Valid @RequestBody CommentRequest commentRequest) {
+        commentRequest.setUser(memberService.getId());
+        return ResponseEntity.ok(commentService.createComment(commentRequest));
+    }
+
+    @PutMapping
+    public ResponseEntity<?> updateComment (@RequestBody CommentRequest commentRequest) {
```

Comment:
```suggestion
    public ResponseEntity<?> updateComment(@RequestBody @Valid CommentRequest commentRequest) {
```
convention 은 위와 같이 맞춰주시구, update 시에도 valid 어노테이션을 붙여주세요~!